chapter06/mssw/file_reader.pm

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/perl
```

Comment:
Sure, but the shell _still_ doesn't need to know how to execute this. If it does, none of the functions are called, so nothing will happen after compilation.